lib/db.js

```
@@ -44,6 +47,10 @@ function createObservationsReplicationStream () {
   return observationsDb.log.replicate()
 }
 
+function wipeDb (db, path, cb) {
+  db.close(() => rimraf(path, () => mkdirp(path, cb)))
```

If `rimraf` fails, its error will be lost.